### ISSUE TITLE
astilectron-bundler -v fails on osx 10.13.4

### ISSUE THREAD

**Issue by zanuka** [asticode/go-astilectron-demo#29]
I followed the demo instructions and all went well until the bundler step. I received the following error:
`FATA[0033] bundling failed: bundling for environment darwin/amd64 failed: building failed: # github.com/asticode/go-astilectron-demo
./bind_darwin_amd64.go:537:34: Asset redeclared in this block
	previous declaration at ./bind.go:3:34
./bind_darwin_amd64.go:728:38: RestoreAssets redeclared in this block
	previous declaration at ./bind.go:7:38
: exit status 2`

**Comment by asticode** [comment]
@zanuka did you run the second command of the [first step](https://github.com/asticode/go-astilectron-demo#step-1-install-the-demo)?

`rm $GOPATH/src/github.com/asticode/go-astilectron-demo/bind.go`

Does this fix your issue?

**Comment by zanuka** [comment]
@asticode yep that was it, I missed that rm step... just got it to install and it generated perfectly. was able to open up the demo app inside /output/darwin-amd64 🏁 

nice work, really like this project
